src/models/StyleSheet.js

```diff
@@ -3,6 +3,28 @@
 /* Wraps glamor's stylesheet and exports a singleton for the rest
 *  of the app to use. */
 
-import { StyleSheet } from '../vendor/glamor/sheet'
+import { StyleSheet as GlamorStyle } from '../vendor/glamor/sheet'
 
-export default new StyleSheet({ speedy: false, maxLength: 40 })
+class StyleSheet {
+  styleSheet: any
```

Comment:
@relekang what should be the whole line be?  Not really familiar with flow.  The `as` in this line is importing it as a different variable, aka `var b = a`.